### PR TITLE
Finished front end functionality.

### PR DESCRIPTION
**Pull request by michaelrtobias**
Every component is working functionally. More tests and some styling are next.
Front End full funtionality  pre styling

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@Bouland95

File: Client/App.jsx [hunk 1]
```diff
@@ -10,37 +10,69 @@ class App extends React.Component {
...
+        mainImageBeforePopOut: ''
       }
   }
 
   changeViewBack() {
     this.setState({
-      mainImageClicked: false
+      mainImageClicked: false,
+      mainImageId: this.state.mainImageBeforePopOut,
+      mainImageIndex: this.state.imageList.indexOf(this.state.imageList.filter((image) => { return image._id === this.state.mainImageBeforePopOut})[0])
```
Comment (Bouland95) [inline Client/App.jsx:23]: Break this into a couple or few lines

File: Client/BottomBar.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import BottomBarEntry from './BottomBarEntry.jsx'
+
+const BottomBar = (props) => (
+      <div className="bottomBar">
+        {props.imageList.map((image) => <BottomBarEntry
```
Comment (Bouland95) [inline Client/BottomBar.jsx:6]: start `<BottomBarEntry` on a new line

File: Client/GreyedOutImage.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+class GreyedOutImage extends React.Component {
+  constructor(props) {
+    super(props)
+
+  }
```
Comment (Bouland95) [inline Client/GreyedOutImage.jsx:6]: I would remove the newline at 5

File: Client/PopOut.jsx [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
+          <div>
+            <Exit changeViewBack={props.changeViewBack}/>
+          </div>
+          <div>
+            <div className="mainCarousel">
+              <LastPicture lastImage={props.lastImage}/>
+              <BigImage mainImage={props.mainImage}/>
+              <NextPicture nextImage={props.nextImage}/>
+            </div>
+            <div>
```
Comment (Bouland95) [inline Client/PopOut.jsx:20]: you have an unnecessary div at line 14

File: Client/PopOutBigImage.jsx [hunk 5]
```diff
@@ -2,18 +2,10 @@ import React from 'react';
...
+const BigImage = (props) => (
 
-  }
-
-
-  render() {
-    return (
-      <img src="https://media2.giphy.com/media/3o6ozfYuxtdUTHnQQ0/giphy.gif" alt="needs-to-be-filled-in"></img>
+      <div className={"bigImageContainer"}>
+        <img className="bigImage" src={props.mainImage.url} alt={props.mainImage.url}></img>
```
Comment (Bouland95) [inline Client/PopOutBigImage.jsx:8]: break this into multiple lines since you did that in mainImage.jsx

**Review by Bouland95 — APPROVED** [review]
Grats on completing the functionality!

**Pull request merged**